gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/gobblin/service/FlowConfigsResource.java

```diff
@@ -115,23 +126,25 @@ public FlowConfig get(ComplexResourceKey<FlowConfigId, EmptyRecord> key) {
       return flowConfig.setFlowGroup(flowGroup).setFlowName(flowName).setProperties(flowPropsAsStringMap);
     } catch (URISyntaxException e) {
       logAndThrowRestLiServiceException(HttpStatus.S_400_BAD_REQUEST, "bad URI " + flowName, e);
-    } catch (JobSpecNotFoundException e) {
+    } catch (SpecNotFoundException e) {
       logAndThrowRestLiServiceException(HttpStatus.S_404_NOT_FOUND, "Flow requested does not exist: " + flowName, null);
     }
 
     return null;
   }
 
   /**
-   * Build a {@link JobSpec} from a {@link FlowConfig}
+   * Build a {@link FlowSpec} from a {@link FlowConfig}
    * @param flowConfig flow configuration
-   * @return {@link JobSpec} created with attributes from flowConfig
+   * @return {@link FlowSpec} created with attributes from flowConfig
    */
-  private JobSpec createJobSpecForConfig(FlowConfig flowConfig) {
+  private FlowSpec createFlowSpecForConfig(FlowConfig flowConfig) {
     ConfigBuilder configBuilder = ConfigBuilder.create()
         .addPrimitive(ConfigurationKeys.JOB_GROUP_KEY, flowConfig.getFlowGroup())
```

Comment:
Should we remove the setting of job_group_key and job_name_key since the flow equivalent is being set?